src/main/java/net/pms/newgui/components/CustomJSpinner.java

```diff
@@ -64,30 +64,30 @@ public CustomJSpinner(boolean enterMoveFocus) {
 	 * {@link #createEditor(SpinnerModel)} method is used to create an editor
 	 * that is suitable for the model.
 	 *
-	 * @param model
+	 * &#64;param model
 	 *            the model (<code>null</code> not permitted).
 	 *
-	 * @throws NullPointerException
-	 *             if <code>model</code> is <code>null</code>.
+	 * @throws NullPointerException if <code>model</code> is <code>null</code>.
 	 */
 	public CustomJSpinner(@Nonnull SpinnerModel model, boolean enterMoveFocus) {
 		super(model);
 		if (model instanceof SpinnerIntModel) {
-			this.addMouseWheelListener(new MouseWheelRoll(
-				this,
-				((SpinnerIntModel) model).getMinimum(),
-				((SpinnerIntModel) model).getMaximum(),
-				((SpinnerIntModel) model).getStepSize()
-			));
+			this.addMouseWheelListener(new MouseWheelRoll(this, ((SpinnerIntModel) model).getMinimum(),
```

Comment:
In this case I think it was better before, so maybe the rule should be changed

Reply:
The same here